src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -106,6 +107,7 @@ object WorkflowActor {
   sealed trait StartMode {
     def runInitialization(actor: WorkflowActor): Future[Unit]
     def start(actor: WorkflowActor): Unit
+    def replyTo: Option[ActorRef]
```

Comment:
That's not me being passive aggressive BTW, it looks like for once maybe I'd say "no" to that.